services/web/client/source/class/osparc/dashboard/StudyBrowser.js

```diff
@@ -594,7 +594,7 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
         this.__startStudy(studyData["uuid"]);
       }, this);
       studyDetails.addListener("updateTags", () => {
-        this.__resetStudyList(osparc.store.Store.getInstance().getStudies());
+        this.reloadUserStudies();
```

Comment:
instead of reloading them all, reload only the one updated:
reloadUserStudy()

Reply:
done